izpack-util/src/main/java/com/izforge/izpack/util/IoHelper.java

```diff
@@ -348,15 +354,34 @@ public static String translatePath(String destination)
     }
 
     /**
-     * Returns the value of the environment variable given by key. This method is a work around for
-     * VM versions which do not support getenv in an other way. At the first call all environment
-     * variables will be loaded via an exec. On Windows keys are not case sensitive.
+     * Returns the value of the environment variable given by key. This method is a workaround for
+     * VM versions which do not support getenv. If it is supported then we will call it (as it
+     * avoids encoding related issue) or else at the first call all environment variables will be
+     * loaded via an exec. On Windows keys are case-insensitive.
      *
      * @param key variable name for which the value should be resolved
      * @return the value of the environment variable given by key
      */
     public static String getenv(String key)
     {
+        Method method = null;
+        try {
+            method = System.class.getDeclaredMethod("getenv", String.class);
+        } catch (NoSuchMethodException ignore) {
+        }
+        if (OsVersion.IS_WINDOWS)
+        {
+            key = key.toUpperCase();
+        }
+        if (method != null)
+        {
+            try {
+                return (String) method.invoke(null, key);
+            } catch (IllegalAccessException | InvocationTargetException e) {
+                logger.log(Level.SEVERE, "Unexpected exception", e);
+                throw new RuntimeException(e);
+            }
+        }
```

Comment:
Why not just use the `System.getenv()` method directly? This method should be available since Java 1.5

Reply:
I think we can directly use `System.getenv(String name)` method, no need to keep cache using `loadEnv()`. The title is as per the issue title.

Reply:
@hiteshbosamiya I then would clean up the code in this way and rename the issue too...

Reply:
sure, thanks

Reply:
@reinhapa, since you said that you will do the changes, I assigned the issue to you. I don't mind making the changes. Just to avoid the confusion, I will make the changes. Please suggest the issue title.

Reply:
I would be happy if you could do the changes. I would suggest to change the issue to something like "Use of default Java environment handling"